src/gui/search/searchwidget.cpp

```diff
@@ -67,7 +71,12 @@
 #include "searchjobwidget.h"
 #include "ui_searchwidget.h"
 
+const int HISTORY_FILE_MAX_SIZE = 10 * 1024 * 1024;
+const int SESSION_FILE_MAX_SIZE = 10 * 1024 * 1024;
+const int RESULTS_FILE_MAX_SIZE = 10 * 1024 * 1024;
+
 const QString DATA_FOLDER_NAME = u"SearchUI"_s;
+const QString HISTORY_FILE_NAME = u"History.txt"_s;
 const QString SESSION_FILE_NAME = u"Session.json"_s;
```

Comment:
Off topic.
I noticed there are more json files now. Would it be better that it is stored with `QJsonDocument::Compact` format?
It could reduce size on disk and deter novice users from tampering it.

Reply:
> Would it be better that it is stored with `QJsonDocument::Compact` format?
> It could reduce size on disk and deter novice users from tampering it.

Well, that makes sense.